diffsync/helpers.py

```diff
@@ -398,11 +406,17 @@ def sync_model(
             elif self.action == "update":
                 if model is None:
                     raise ObjectNotUpdated(f"Failed to update {self.model_class.get_type()} {ids} - not found!")
-                model = model.update(attrs=attrs)
+                if model.model_flags & DiffSyncModelFlags.CRUD_NO_UPDATE:
+                    model = model.update_base(attrs=attrs)
```

Comment:
`update_base` still updates the DiffSyncModel instance with the new `attrs`? To me that doesn't align with the concept of `CRUD_NO_UPDATE` as a label. I would expect that it would make no changes to the DiffSyncModel at all... consider also the case of repeated syncing/diffing with these flags in play:

- `diff()` - shows model delta in the diffs, even the models that set CRUD_NO_*
- `sync()` - calls `_base` methods on all models, resulting in DiffSyncModel instances being created/updated/deleted
- `diff()` - now shows no diffs at all, even on the CRUD_NO_* models (because they were modified by `sync()`)

That seems wrong to me.

Reply:
Good point, I've updated the logic to not touch the local object as well